Description: Remove a test that segfaults or times out with giac 1.6.0.7
Author: Tobias Hansen <thansen@debian.org>
Bug: https://trac.sagemath.org/ticket/30537

--- a/sage/src/sage/rings/polynomial/multi_polynomial_ideal.py
+++ b/sage/src/sage/rings/polynomial/multi_polynomial_ideal.py
@@ -2096,13 +2096,6 @@
             Ideal (y^2 - x*z, x*y - z, x^2 - y) of Multivariate
             Polynomial Ring in x, y, t, s, z over Rational Field
 
-        You can use Giac to compute the elimination ideal::
-
-            sage: I.elimination_ideal([t, s], algorithm="giac") == J
-            ...
-            Running a probabilistic check for the reconstructed Groebner basis...
-            True
-
         The list of available Giac options is provided at
         :func:`sage.libs.giac.groebner_basis`.
 
@@ -2124,9 +2117,6 @@
             sage: J = I.elimination_ideal([t,s]); J
             Ideal (y^2 - x*z, x*y - z, x^2 - y) of Multivariate
             Polynomial Ring in x, y, t, s, z over Algebraic Field
-            sage: I.elimination_ideal([t, s], algorithm="giac") == J
-            Running a probabilistic check for the reconstructed Groebner basis...
-            True
         """
         if not isinstance(variables, (list, tuple)):
             variables = (variables,)
